docs/ref/middleware.txt

```diff
@@ -471,3 +471,8 @@ Here are some hints about the ordering of various Django middleware classes:
 #. :class:`~django.contrib.redirects.middleware.RedirectFallbackMiddleware`
 
    Should be near the bottom as it's a last-resort type of middleware.
+
+#. :class:`~django.middleware.security.SecurityMiddleware`
+
+   After ``GZipMiddleware``: ``SecurityMiddleware`` may access request and
```

Comment:
I don't think this is currently true; `SecurityMiddleware` doesn't do anything with request or response bodies at this point, only headers. So I don't think its position relative to `GZipMiddleware` actually matters.

I guess it's _possible_ that we might introduce something into `SecurityMiddleware` in the future that needs access to response content; doesn't seem likely to me, but it's possible.